.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules
 .idea
 *.swp
+.nyc_output
+.vscode
```

Comment:
Nit: please add a a newline at the end of the file.

Reply:
Done!